shared/reducers/tracker.js

```diff
@@ -512,7 +512,10 @@ function stateToColor (state: SimpleProofState): string {
   return 'gray'
 }
 
-function proofStateToSimpleProofState (proofState: ProofState, diff: ?TrackDiff, remoteDiff: ?TrackDiff): ?SimpleProofState {
+function proofStateToSimpleProofState (proofState: ProofState, diff: ?TrackDiff, remoteDiff: ?TrackDiff, breaksTracking: boolean): ?SimpleProofState {
```

Comment:
I spoke to @maxtaco and have ticketed making the gui not understand all these nuances which'll hopefully close the door on these types of mismatches between the CLI and the gui.